src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -292,7 +288,7 @@ InferenceEngine::Parameter ExecNetwork::GetMetric(const std::string &name) const
     const auto& graph = graphLock._graph;
     const auto& config = graph.getConfig();
 
-    if (isLegacyAPI()) {
+    if (!_cfg.isNewApi) {
```

Comment:
This line should be the minimum change.
Once ov::core is instantiated and passed to CPU plugin, api version should have been decided, it will not change any more.